### PR TITLE
feat(components): [cascader] update cascader component to expost presentText

### PR DESCRIPTION
**Pull request by 0song**
Please make sure these boxes are checked before submitting your PR, thank you!

This will expose `presentText`, to keep the same expose logic as Element UI2. 

So that user can access the component selected text to increase accessibility.

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/cascader.md [hunk 1]
```diff
@@ -188,6 +188,7 @@ cascader/panel
 | cascaderPanelRef              | cascader panel ref                                                                                                | ^[object]`ComputedRef<any>`                                     |
 | togglePopperVisible ^(2.2.31) | toggle the visible type of popper                                                                                 | ^[Function]`(visible?: boolean) => void`                        |
 | contentRef                    | cascader content ref                                                                                              | ^[object]`ComputedRef<any>`                                     |
+| presentText                    | selected content text                                                                                              | ^[string]                                     |
```
Comment (makedopamine) [inline docs/en-US/component/cascader.md:191]:
```suggestion
| presentText ^(2.8.4)                   | selected content text                                                                                              | ^[object]`ComputedRef<string>`                                     |
```

Comment (0song) [reply docs/en-US/component/cascader.md]: fixed, thanks

File: docs/en-US/component/cascader.md [hunk 2]
```diff
@@ -188,6 +188,7 @@ cascader/panel
 | cascaderPanelRef              | cascader panel ref                                                                                                | ^[object]`ComputedRef<any>`                                     |
 | togglePopperVisible ^(2.2.31) | toggle the visible type of popper                                                                                 | ^[Function]`(visible?: boolean) => void`                        |
 | contentRef                    | cascader content ref                                                                                              | ^[object]`ComputedRef<any>`                                     |
+| presentText                   | selected content text                                                                                              | ^[object]`ComputedRef<string>`                                 |
```
Comment (btea) [inline docs/en-US/component/cascader.md:191]:
```suggestion
| presentText  ^(2.8.4)         | selected content text                                                                                              | ^[object]`ComputedRef<string>`                                 |
```
Maybe we should add a version tag.

Comment (iamkun) [reply docs/en-US/component/cascader.md]: fixed, thanks

Comment (0song) [reply docs/en-US/component/cascader.md]: fixed, thanks

**Review by btea — APPROVED** [review]

**Pull request merged**